sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusTestEnvironment.cs

```diff
@@ -28,30 +28,9 @@ public ServiceBusTestEnvironment() : base("servicebus")
         /// </summary>
         public static ServiceBusTestEnvironment Instance { get; } = new ServiceBusTestEnvironment();
```

Comment:
Is this still needed?

Reply:
Yes, we still need to have a single instance of `ServiceBusTestEnvironment` somewhere.